### DISCUSSION TITLE
Many things don't work. Eg, can't find any media output device, plz help

### DISCUSSION THREAD

**Discussion by lVentus** [end-4/dots-hyprland#504] [Q&A]
no audio device found. When I use aylur's dot, the audio worked perfect. And the indicators and screenshot don't work at all, either. Gemini also  doesn't answer anything.Did I miss something?

**Comment by clsty** [comment]
- Have you rebooted after installing the script?
- If rebooting does not solve your problem, run `pkill ags; ags` and post the output here.

**Reply by lVentus** [reply]
I didn’t use the install script, I am a Nixos user. But I installed all pkgs in cirnos and also all pkgs found here in the script/dependencies.conf. I am not sure what is missing

**Comment by end-4** [comment]
> the indicators and screenshot don't work at all

perhaps you forgot to add the required packages to ags part of your flake? maybe see [my config](https://github.com/end-4/CirnOS/blob/24a79b1b371c77ff7f8e6584b8551dbe67612b6c/homes/end/ags.nix#L17-L35) as a reference

> Gemini also doesn't answer anything

make sure you have a valid key. send the command `/key` for instructions

**Reply by lVentus** [reply]
I think the indicators is not the important part. the audio device part don’t work. Now, I can’t get any devices in right side system panel. But I do can get them in gnome settings and the pavucontrol gui. The bind  in hyprland also works well. So I don’t know why. I am using pipewire with pulse = true. 

The screenshot part, I will run bash and check the log later.

Solved Gemini problem!

thanks for your reply

**Reply by end-4** [reply]
you'll have to stick to pavucontrol for now
the audio device feature was added way after i stopped using nixos
unless you can provide logs... run `pkill ags; ags` like clsty suggested (or whatever the equivalent in nixos is)

**Reply by lVentus** [reply]
actually， there is almost no useful info in logs. Only one error at beginning : no window named overview.(but overview works perfect). After that there is only one warning “ can’t assign WiFi xxxx, it is not icon or file. But the WiFi icon shows normally. There are no other infos in log
